_book-template/0-2-copyright.md

```diff
@@ -26,4 +26,4 @@ This licence means you may share, copy and redistribute the material in any medi
 * Non-Commercial — You may not use the material for commercial purposes.
 * No Derivatives — If you remix, transform, or build upon the material, you may not distribute the modified material.
 
-For any other uses, please contact [Electric Book Works](http://electricbookworks.com).
+For any other uses, please contact [[Bettercare](http://bettercare.co.za).
```

Comment:
@nkoeries Double bracket here. I've done a very rough spot check, so you'll need to do a search if this is copy-pasted in various places.

Reply:
@arthurattwell fixed.